test/jdk/java/util/zip/DataDescriptorSignatureMissing.java

```diff
@@ -26,122 +26,78 @@
  * @bug 8056934
  * @summary Check ability to read zip files created by python zipfile
  * implementation, which fails to write optional (but recommended) data
- * descriptor signatures.  Repro scenario is a Java -> Python -> Java round trip:
- * - ZipOutputStream creates zip file with DEFLATED entries and data
- *   descriptors with optional signature "PK0x0708".
- * - Python reads those entries, preserving the 0x08 flag byte
- * - Python outputs those entries with data descriptors lacking the
- *   optional signature.
- * - ZipInputStream cannot handle the missing signature
- *
+ * descriptor signatures.
```

Comment:
Hello Eirik, I think the summary should no longer have references to python. Since this test was introduced for https://bugs.openjdk.org/browse/JDK-8056934, perhaps we can just change the `@summary` of this test definition to say:
>
>@summary Verify the ability to read zip files whose local header data descriptor is missing the optional signature

Reply:
Thanks, I updated the summary according to your suggestion. I left it there to keep the history/context of the test, but I guess that's a bit detailed and is already captured in the bug.